modules/va_facilities/README.yml

```diff
@@ -100,7 +100,8 @@ paths:
       parameters:
         - name: ids
           description: |
-            List of command separated ids of facilities to retrieve in a single request
+            List of command separated ids of facilities to retrieve in a single request.
```

Comment:
command -> comma ?

Reply:
Oh that's an interesting one. I should indeed be `comma`.